cookbooks/aws-parallelcluster-slurm/files/default/head_node_slurm/slurm/pcluster_slurm_config_generator.py

```diff
@@ -212,6 +218,16 @@ def _vcpus(compute_resource) -> int:
     return vcpus_count if not disable_simultaneous_multithreading else (vcpus_count // threads_per_core)
 
 
+def _realmemory(compute_resource) -> int:
+    """Get the RealMemory parameter to be added to the Slurm compute node configuration."""
+    instance_type = compute_resource["InstanceType"]
+    instance_type_info = instance_types_data[instance_type]
+    memory_info = instance_type_info.get("MemoryInfo", {})
+    ec2_memory = memory_info.get("SizeInMiB")
+    realmemory = math.floor(ec2_memory * 0.95)
```

Comment:
I suggest to avoid magic numbers. Let's define a constant with value `0.95`, having a self-explanatory name and if necessary a comment.

Reply:
Discussed about it with @demartinofra. We agreed on having the 0.95 threshold defined in the attributes for the reasons above.

Reply:
Agreed. I will add it in a future PR.